lte/gateway/c/oai/lib/n11/SubscriberdbServiceClient.h

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <grpc++/grpc++.h>
+#include "lte/protos/subscriberauth.grpc.pb.h"
+#include "lte/protos/subscriberauth.pb.h"
+#include "GRPCReceiver.h"
+#include <stdint.h>
+#include <functional>
+#include <memory>
+#include "3gpp_38.413.h"
+
+using grpc::Status;
+using magma::GRPCReceiver;
+using magma::lte::M5GAuthenticationInformationAnswer;
+using magma::lte::M5GAuthenticationInformationRequest;
+using magma::lte::M5GSubscriberAuthentication;
+
+namespace magma5g {
+
+class SubscriberdbServiceClient {
```

Comment:
change the file name and class name to be more specific about the subcriberdb service being consumed.

Reply:
Comments addressed.